packages/perseus/src/widgets/dropdown/dropdown-validator.test.tsx

```diff
@@ -1,15 +1,20 @@
 import dropdownValidator from "./dropdown-validator";
 import {question1} from "./dropdown.testdata";
 
-import type {PerseusDropdownUserInput} from "../../validation.types";
+import type {
+    PerseusDropdownRubric,
+    PerseusDropdownUserInput,
+} from "../../validation.types";
 
 describe("dropdownValidator", () => {
     it("returns invalid for user input of 0", () => {
         // Arrange
         const userInput: PerseusDropdownUserInput = {
             value: 0,
         };
-        const rubric = question1.widgets["dropdown 1"].options;
```

Comment:
This was still passing, but it was still showing rubric as PerseusDropdownWidgetOptions. Updated the tests to be as specific as the rubric type.